_targets.R

```diff
@@ -25,29 +25,34 @@ p1_targets_list <- list(
     error = 'continue'
   ),
   tar_target(
-    site_data,
-    bind_rows(site_data_frames),
+    nwis_data_csv,
+    {out_file <- '2_process/out/site_data.csv'
+      write_csv(bind_rows(site_data_frames), file = out_file)
+      return(out_file)
+      }
```

Comment:
This is also what I did when I switched to a file. Didn't feel like writing a custom function. I do wish that `write_csv()` would just return the filepath ... I might actually go make an issue request for that as an option and link to `targets` as an example because it bothers me so much! Would be much less coding in targets pipelines if that worked.

Reply:
LOL I did ... https://github.com/tidyverse/readr/issues/1282 😄 I doubt they will implement it, though

Reply:
Yeah, that would be nice.  Seems more useful than invisibly returning the data frame like it does now.  None of the major csv writers seem to return that path though 😞